src/views/AccessControl/LocalUserManagement/TableRoles.vue

```diff
@@ -35,80 +35,80 @@ export default {
       items: [
         {
           description: "Configure components managed by this service",
-          admin: true,
+          administrator: true,
           operator: false,
-          user: false,
+          readonly: false,
           callback: false
```

Comment:
This got renamed to NoAccess. 
Need a master build from the last month to see this. 

curl -k https://${bmc}/redfish/v1/AccountService/Roles
{
  "@odata.context": "/redfish/v1/$metadata#RoleCollection.RoleCollection",
  "@odata.id": "/redfish/v1/AccountService/Roles",
  "@odata.type": "#RoleCollection.RoleCollection",
  "Description": "BMC User Roles",
  "Members": [
    {
      "@odata.id": "/redfish/v1/AccountService/Roles/Administrator"
    },
    {
      "@odata.id": "/redfish/v1/AccountService/Roles/Operator"
    },
    {
      "@odata.id": "/redfish/v1/AccountService/Roles/ReadOnly"
    },
    {
      "@odata.id": "/redfish/v1/AccountService/Roles/NoAccess"
    }

Reply:
Done